l10n_br_hr_contract/models/hr_contract_labor_resignation_cause.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016  Daniel Sadamo - KMEE Informática
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models, fields, api
+
+
+class HrContractResignationCause(models.Model):
+    _name = 'hr.contract.resignation.cause'
+
+    name = fields.Char(string='Resignation cause')
```

Comment:
This class must have at least one mandatory field to prevent an empty record creation. (In this case I think name and code must be required, right?)